passes/source2il.nim

```diff
@@ -21,6 +21,9 @@ type
   NodeSeq    = seq[Node]
 
   EntityKind = enum
+    ekNone        ## signals "non-existent"
+    ekBuiltin     ## some built-in special entity
```

Comment:
```suggestion
    ekBuiltinVal     ## some built-in special value
```

I think it might be better to be specific and then have a catch-all in the future if we need it.

Reply:
Thanks, that's a better approach.